src/components/InputMobileNumber.js

```diff
@@ -217,7 +217,7 @@ class InputMobileNumber extends Component {
 
   render() {
     return (
-      <SContainer>
+      <SContainer {...this.props}>
```

Comment:
not sure what you mean, its using the spread operator. unless you mean destruct it firs `const = { ...props } = this.props` ??